traffic_ops/testing/api/v14/todb.go

```diff
@@ -103,6 +103,12 @@ func SetupTestData(*sql.DB) error {
 		os.Exit(1)
 	}
 
+	err = SetupServerCapabilities(db)
+	if err != nil {
+		fmt.Printf("\nError setting up server capabilities %s - %s, %v\n", Config.TrafficOps.URL, Config.TrafficOps.Users.Admin, err)
+		os.Exit(1)
```

Comment:
I know this PR is just following the established pattern here, but this function can never return a non-nil error since it calls os.Exit whenever it gets an error from the functions it calls. Perhaps something to note with a TODO comment to follow up on later.